### DISCUSSION TITLE
A minor contribution of public domain logos

### DISCUSSION THREAD

**Discussion by ineuw** [linuxmint/discussions#77] [Development]
I added [these Pluma and a single Xed logo](https://commons.wikimedia.org/wiki/Category:Alternative_logos_of_free_software) for users with difficulty identifying apps in the menu or the panel. I use them wherever I install Linux Mint. These are in the public domain, which means that they don't have to be attributed, (especially, if they are not liked).

I've spent minimal time on the **Xed** logo, because I switched to Pluma. I don't like the logo, but will have to do until the need rises to come up with something more imaginative.

I also created a **LibreOffice** logo to be quickly identified in the panel to replace their indistinguishable white logo, but I must verify the legality of uploading the logo. The logo itself is ugly, but noticeable.

The images were created with Irfanview in Wine and the text characters in LibreOffice.

**Comment by Secret-chest** [comment]
Created a commons category just for you?

If you want LM to use these, I'm sorry to say they probably won't. Modern icons are vectors and don't have text like that.

**Reply by ineuw** [reply]
Thanks. In the longer term, .svg is the final goal. Until then . . . .  

P.S. The category is my creation. I am a long standing contributor to Wikimedia.
P.P.S. I didn't expect them to be used by LM officially, I am just advertising their existence to users challenged by app identification.
